app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Api::V1::CommentsController < ApplicationController
+    before_action :authenticate_user!, :except => [:show, :index]
+    
+    def index
+        @comments = Comment.find(params[:recipe_id])
```

Comment:
I think this would find comments where `comments.id = :recipe_id`, which is probably not what you want. I think probably `Comment.where(recipe_id: params[:recipe_id]).all` is what you want.